vms/event/services.py

```diff
@@ -12,6 +14,21 @@ def event_not_empty(event_id):
     return result
 
 
+def get_event_by_shift_id(shift_id):
```

Comment:
@vubo Can you add a comment before this function, that what this can be used and specify that it may be useful in the future. What happens is, sometimes new contributors would like to start with cleaning the already existing code, and a function which is not used anywhere becomes a victim. If we add a comment we all will remember, that yes it is not used anywhere, right now, but can be used in the future, if needed.